lib/middleware/serveResources.js

```diff
@@ -6,6 +6,8 @@ import fresh from "fresh";
 
 const rProperties = /\.properties$/i;
 const rReplaceVersion = /\.(library|js|json)$/i;
+const rManifest = /\/manifest.json$/i;
+const rResourcesPrefix = /^\/resources/i;
```

Comment:
This should only match for `/resources/` but not for `/resources-something-else/`, right?
```suggestion
const rResourcesPrefix = /^\/resources\//i;
```

Reply:
Done